source/imaer-util/src/main/java/nl/overheid/aerius/validation/RblCohesionValidator.java

```diff
@@ -62,12 +63,20 @@
     private final List<AeriusException> errors = new ArrayList<>();
     private final List<AeriusException> warnings = new ArrayList<>();
 
-    void add(final ImportParcel importResult) {
-      addSources(importResult);
-      addPoints(importResult);
-      addMeasures(importResult);
-      addDispersionLineIds(importResult);
-      this.corrections.addAll(importResult.getSituation().getNslCorrections());
+    void add(final ScenarioSituation situation) {
+      addSources(situation);
+      addMeasures(situation);
+      addDispersionLineIds(situation);
+      this.corrections.addAll(situation.getNslCorrections());
+    }
+
+    public void addPoints(final List<CalculationPointFeature> calculationPoints) {
```

Comment:
Don't really see a point in making this public since the class is private anyway? Can't hurt though I guess.

Reply:
As you mentioned the whole class is private so has no impact. To keep all methods the same I've added the `public` to the `add` method.  Other option would be to remove `public` from all other methods.